assets/sass/imports/_sponsors.scss

```
@@ -0,0 +1,10 @@
+.sponsor-container {
+    padding-top: 80px;
```

```suggestion
    margin-top: 80px;
```
For external spacing margin is preferred.